src/mediapipe_calculators/pyobject_ovtensor_converter.cc

```diff
@@ -0,0 +1,119 @@
+//*****************************************************************************
+// Copyright 2023 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+#include <unordered_map>
+
+#include "../precision.hpp"
+#include "../mediapipe_internal/pythonnoderesource.hpp"
+#include "../python/ovms_py_tensor.hpp"
+#include <openvino/openvino.hpp>
+
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wdeprecated-declarations"
+#include "mediapipe/framework/calculator_framework.h"
+#pragma GCC diagnostic pop
+#include "src/mediapipe_calculators/pyobject_ovtensor_converter.pb.h"
+
+#include <pybind11/embed.h>  // everything needed for embedding
+#include <pybind11/stl.h>
+
+#include "../python/python_backend.hpp"
+
+namespace py = pybind11;
+using namespace py::literals;
+using namespace ovms;
+
+namespace mediapipe {
+
+class PyobjectOvtensorConverterCalculator : public CalculatorBase {
+    mediapipe::Timestamp outputTimestamp;
+public:
+    static absl::Status GetContract(CalculatorContract* cc) {
+        LOG(INFO) << "PyobjectOvtensorConverterCalculator [Node: " << cc->GetNodeName() << "] GetContract start";
+        RET_CHECK(!cc->Inputs().GetTags().empty());
+        RET_CHECK(!cc->Outputs().GetTags().empty());
+        RET_CHECK(cc->Inputs().GetTags().size() == 1);
+        RET_CHECK(cc->Outputs().GetTags().size() == 1);
+        RET_CHECK(*(cc->Inputs().GetTags().begin()) == "OVTENSOR" || *(cc->Inputs().GetTags().begin()) == "OVMS_PY_TENSOR");
```

Comment:
We could write 1 check (L49-50) so that we are sure that both tags are set correctly)
If input is OVtensor then output must be pytensor
else the other way arround. It would be probably cleaner to use if else in L52 to include check there - if input is one then we expect the other tag on output.